### DISCUSSION TITLE
out of memory after upgrading to 0.21.3

### DISCUSSION THREAD

**Discussion by ghost** [aquasecurity/trivy#1495] [Q&A] [answered]
hi,
after upgrading to 0.21.3 we have some out of memory errors during image scans.
(previous version was 0.16.0 and upgrade to 0.17.x also gives out of memory)
free memory is about 1.2Gb 
what are the minimal requirements for the new version?
thanks in advance,
  best regards
   noirabys

**Comment by ghost** [comment, marked as answer]
after assigning more ram to the instance it looks like it takes round about
2,6 Gb now.

**Reply by knqyf263** [reply]
Could you tell me which image you scanned? We'd reproduce it.
